posthog/session_recordings/test/test_factory.py

```diff
@@ -60,37 +61,46 @@ def create_session_recording_events(
     # If not given we will create a mock full snapshot
     snapshots: Optional[List[dict]] = None,
     chunk_size: Optional[int] = 512 * 1024,
-) -> List[str]:
-
-    if window_id is None:
-        window_id = session_id
+    use_replay_table: bool = True,
```

Comment:
these two parameters let us switch a test from only recording events, only replay events or both. so that we can find things that won't work without recording events